.pre-commit-config.yaml

```diff
@@ -141,3 +141,8 @@ repos:
           '--disable',
           'dangling-hyphen,line-too-long',
         ]
+  - repo: https://github.com/dnephin/pre-commit-golang
```

Comment:
It seems that this project isn't maintained: https://github.com/dnephin/pre-commit-golang/issues/98

Can we use other repo?

Reply:
yes, changed to golangci-lint to match arrow-adbc, but disabled most options

Reply:
Oh nice catch!